feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -1,14 +1,31 @@
+import { runImporter } from "../importer";
+
 export default class DataAccount {
     constructor() {
-        this.importingResults = [];
+        this.importingReports = [];
         this.importedFileNames = [];
-        this.personalData = { name: { givenName: "", familyName: "" } };
-        this.preferredLanguage = [];
         this.analyses = {};
         this.reports = {};
         this.analysesExecutionResults = [];
     }
 
+    async import({ importers, zipFile, pod }) {
```

Comment:
This was already in #1190, right? Wouldn't it be better to put some foundation work, such as this, in a small PR so that we can get it out of the way?